### PR TITLE
Obtain library docstrings

### LINKED ISSUE #444: Library tab - Parser docstring parser
Parse python files extract doc string and convert to html

### PR DESCRIPTION
**Pull request by jkfran**
## Done

Parse python files to obtain their docstrings

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Mocked data is under http://localhost:8045/<charm>/libraries

## Issue / Card

Fixes #444

### CODE REVIEW

File: templates/details/libraries.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{% set current_tab = "history" %}
```
Comment (solazio) [inline templates/details/libraries.html:1]:
```suggestion
{% set current_tab = "libraries" %}
```

**Review by solazio — COMMENTED** [review]
but what about the side navigation content?

**Comment by jkfran** [comment]
> but what about the side navigation content?

Yeah, the content we need is there. Sorry for not working more on the template, as an quick example:
```html
    {% for group in libraries.keys() %}
    <div>{{ group }}</div>
      {% for lib in libraries[group] %}
      <div>- {{ lib["name"] }}</div>
      {% endfor %}
    {% endfor %}
```

**Review by solazio — APPROVED** [review]
Looks good :+1:. Thanks

**Pull request merged**